core/src/main/java/com/oath/oak/OakAttachedWriteBuffer.java

```diff
@@ -13,13 +13,14 @@
  * An instance of this buffer is only used when the write lock of the key/value referenced by it is already acquired.
  * This is the reason no lock is acquired in each access.
  */
-class OakAttachedWriteBuffer extends OakAttachedReadBuffer implements OakWriteBuffer, OakUnsafeDirectBuffer {
+final class OakAttachedWriteBuffer extends OakAttachedReadBuffer implements OakWriteBuffer, OakUnsafeDirectBuffer {
```

Comment:
Is it that any class that none inherits from it, is requested to be final?

Reply:
No. Just classes that have only private constructors. There isn't a way to inherit from a class with a private constructor, so it should be final.